config/initializers/loa.rb

```diff
@@ -25,8 +25,8 @@ module LOA
   # DSLogon accounts with these assurance levels are considered LOA3
   DSLOGON_PREMIUM_VERIFIED = %w[2 3].freeze
 
-  IDME_MHV_LOA1 = 'myhealthevet'
```

Comment:
Oh yeah, these have to stay at the longform `myhealthevet`, since we send these loas to id.me as scope parameters